Automattic/jetpack
# AI Client: remove unused image library

Since this lib is not being used, let's clean it up for now. We can add new libs when it's required.

Fixes #

## Proposed changes:

*  AI Client: remove unused image library

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Just confirm that the building process works as expected.
* Confirm the client still provides the resources, by using a Jetpack site and some AI features.
